.golangci.json

```diff
@@ -1,8 +1,7 @@
 {
   "linters": {
     "disable": [
-      "errcheck",
-      "unused"
+      "errcheck"
```

Comment:
Enabling `unused`. I enabled `errcheck` and `forcetypeassert` temporarily while working on this changeset, but they're going to require some chipping before we can enable them globally.